static/app/views/issueDetails/eventNavigation.spec.tsx

```diff
@@ -0,0 +1,85 @@
+import {EventFixture} from 'sentry-fixture/event';
+import {GroupFixture} from 'sentry-fixture/group';
+
+import {render, screen, userEvent} from 'sentry-test/reactTestingLibrary';
+
+import {browserHistory} from 'sentry/utils/browserHistory';
+import * as useMedia from 'sentry/utils/useMedia';
+import EventNavigation from 'sentry/views/issueDetails/eventNavigation';
+
+describe('EventNavigation', () => {
```

Comment:
Probably want some coverage for the jump to element logic?

Reply:
will add tests for making sure that area renders correctly - hard to test this jump logic without the other components of the page but once we have integrated things more, will add tests for that